components/build-templates/base/rbac/build-admin.yaml

```diff
@@ -0,0 +1,46 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: build-admin
+  namespace: build-templates-e2e
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
+      - delete
+
+  - apiGroups:
+      - ''
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: build-admins
+  namespace: build-templates-e2e
+subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: Build Admins team
```

Comment:
Are there any QEs in this group? If not, should we add them as well?

Reply:
Nobody from QE is there, they would have to create a new group in our github organization. But the question is also who owns this namespace? From the history, it seems only Michal took care of it.

Reply:
But I believe we can resolve this later.

Reply:
I see, OK let's leave that for some other time